adapters/onebot/src/bot/message.ts

```diff
@@ -43,17 +44,28 @@ export class OneBotMessenger extends Messenger<BaseBot> {
     }
 
     // flush
-    if (!this.children.length) return
-    const { type, author } = this.stack[0]
+    const { type, data } = this.stack[0]
+    if (!this.children.length && typeof data !== 'number') return
```

Comment:
Isn't the author can be only an `Author` or `string`? Where the hat that `number` from?